api/src/pcapi/utils/requests.py

```diff
@@ -39,16 +39,16 @@ def _redact_url(url: str) -> str:
     return re.sub("api_token=[^&^$]+", "api_token=[REDACTED]", url)
 
 
-def _wrapper(request_func: Callable, method: str, url: str, log_at_error_level=True, **kwargs: Any) -> Response:  # type: ignore [no-untyped-def]
+def _wrapper(request_func: Callable, method: str, url: str, log_at_warning_level=True, **kwargs: Any) -> Response:  # type: ignore [no-untyped-def]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Ajouter `log_at_warning_level: bool=True,‏` suffit à retirer le ` # type: ignore`